traffic_router/core/src/main/java/org/apache/traffic_control/traffic_router/core/router/StatTracker.java

```diff
@@ -452,11 +462,20 @@ public void initialize(final Map<String, List<String>> initMap, final CacheRegis
 
 					t.setRouteType(rt, dsName.toString());
 					t.setResult(ResultType.INIT);
-					t.end();
 
 					saveTrack(t);
 				}
 			}
 		}
 	}
+
+	private void resetHttpStatsFromOverflow() {
+		totalHttpCount = 0;
+		totalHttpTime = 0;
+	}
+
+	private void resetDnsStatsFromOverflow() {
+    	totalDnsCount = 0;
+    	totalDnsTime = 0;
```

Comment:
these lines are missing an indentation

Reply:
actually, this indentation is using spaces instead of tabs, and it appears this is a tab-indented file